website_login_redirect_back/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Ildar Nasyrov <iledarn@gmail.com.com>
+# License MIT (https://opensource.org/licenses/MIT).
+{
+    "name": "Website Login Redirect Back",
```

Comment:
Аналогично в других местах
```suggestion
    "name": "Login Page Redirection",
```